cmd/observer/node_builder/observer_builder.go

```diff
@@ -766,7 +799,11 @@ func (builder *ObserverServiceBuilder) enqueueConnectWithStakedAN() {
 
 func (builder *ObserverServiceBuilder) attachRPCEngine() {
 	builder.Component("RPC engine", func(node *cmd.NodeConfig) (module.ReadyDoneAware, error) {
-		proxy, err := apiservice.NewFlowAPIService(builder.bootstrapIdentities, builder.apiTimeout)
+		ids := builder.upstreamIdentities
+		if len(ids) == 0 {
+			ids = builder.bootstrapIdentities
+		}
```

Comment:
we can't use `bootstrapIdentities` since it will point to the libp2p interface. I think the default should just be an empty list which mean no proxying.

Reply:
Fixed: https://github.com/GetElastech/flow-go/commit/e867cb36f4fde512a7b0cd4d15f6755a46473939